extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java

```diff
@@ -31,7 +31,8 @@ public class WebJarUtil {
 
     private static final Logger LOG = Logger.getLogger(WebJarUtil.class);
 
-    private static final String CUSTOM_MEDIA_FOLDER = "META-INF/branding/";
+    private static final String CUSTOM_MEDIA_FOLDER_MVN = "META-INF/branding/";
+    private static final String CUSTOM_MEDIA_FOLDER_GRADLE = "META-INF/resources/branding/";
```

Comment:
It it expected that the structure of the jar changes when using another build tool? I would have expected the conventions to be the same?

/cc @aloubyansky @snazy @glefloch

Reply:
Yea - that should definitely not be the case.
Quarkus (even old Quarkus 2 apps built via Gradle) pulls web-resources from `META-INF/resources/` ([our web-resources](https://github.com/projectnessie/nessie-ui/blob/e75149978e5884533d151a2176c2f78fdb4a5c04/nessie-ui/build.gradle.kts#L28)), but branding-stuff works fine [in `META-INF/branding` for us](https://github.com/projectnessie/nessie/tree/main/servers/quarkus-server/src/main/resources/META-INF/branding).
Maybe the Gradle-build for the app is wrong?

Reply:
I don't know enough about Gradle, can you check the reproducer shared in the issue ?

Reply:
hehe - the issue is in the issue description - the the 1st screenshot shows `resources/META-INF/resources/branding/logo.png` - that should be `.../META-INF/resources/branding/logo.png`